specification/servicebus/resource-manager/common/v1/definitions.json

```diff
@@ -189,6 +189,59 @@
         }
       },
       "description": "The resource management error additional info."
+    },
+    "systemData": {
```

Comment:
>systemData [](start = 5, length = 10)

Same comment as in the other PR. Recommend referring to common definitions "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/..."